test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -154,25 +191,44 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, func() {
 	})
 
 	Context("Verification of cgroup layout on the worker node", func() {
+		var ctx context.Context = context.TODO()
+		var totalNumberofCgroupMembership int
+		var cgroupProcs, cgroupCpusetCpus, cgroupLoadBalance string
+		BeforeAll(func() {
+			pids, err := ovsPids(ctx, ovsSystemdServices, workerRTNode)
+			Expect(err).ToNot(HaveOccurred(), "unable to fetch pid of ovs services")
+			cmd := []string{"cat", fmt.Sprintf("/rootfs/proc/%s/cgroup", pids[0])}
+			out, err := nodes.ExecCommandOnMachineConfigDaemon(context.TODO(), workerRTNode, cmd)
+			Expect(err).ToNot(HaveOccurred())
+			controllers := bytes.Split(out, []byte("\n"))
+			Expect(len(controllers)).ToNot(Equal(0), "Unable to fetch cgroup membership of %s pid", pids[0])
+			totalNumberofCgroupMembership = len(controllers)
+			ovsSliceCgroup, err = cgroup.PidParser(out)
+			Expect(err).ToNot(HaveOccurred())
+			cgroupSlices := strings.Split(ovsSliceCgroup, "/")
+			parentCgroup := strings.Join(cgroupSlices[:len(cgroupSlices)-1], "/")
+			// This checks if it's cgroupv1 or v2, on v2 the number on
+			// controllers are not available on each line, since v2 has only single hierarchy
+			if len(controllers) > 2 {
```

Comment:
see if this needs to change as well according to @MarSik's  [comment](https://github.com/openshift/cluster-node-tuning-operator/pull/1011/files#r1559140030)